src/views/tickets.js

```diff
@@ -0,0 +1,163 @@
+import paginationFactory from 'react-bootstrap-table2-paginator';
+import UserContext from '../UserContext';
+import { Link } from "react-router-dom"
+import Accordion from '../components/Accordion';
+import * as axios from 'axios';
+
+const columns = [{
+    dataField: 'name',
+    text: 'Ticket',
+    headerStyle: () => {
+      return { width: "20%" };
+    }
+  }, {
+    dataField: 'propertyManager',
+    text: 'Sender',
+    sort: true,
+    headerStyle: () => {
+      return { width: "20%" };
+    }
+  }, {
+    dataField: 'address',
+    text: 'Assigned To',
+    sort: true,
+    headerStyle: () => {
+      return { width: "20%" };
+    }
+  }, {
+    dataField: 'tenants',
+    text: 'Created',
+    sort: true,
+    headerStyle: () => {
+      return { width: "20%" };
+    }
+  }, {
+    dataField: 'dateAdded',
+    text: 'Updated',
+    sort: true,
+    headerStyle: () => {
+      return { width: "20%" };
+    }
+  }];
+
+const pageButtonRenderer = ({
+  page,
+  active,
+  disable,
+  title,
+  onPageChange
+}) => {
+  const handleClick = (e) => {
+    e.preventDefault();
+    onPageChange(page);
+  };
+  const activeStyle = {};
+  if (active) {
+    activeStyle.backgroundColor = 'black';
+    activeStyle.color = 'white';
+  } else {
+    activeStyle.backgroundColor = 'gray';
+    activeStyle.color = 'black';
+  }
+  if (typeof page === 'string') {
+    activeStyle.backgroundColor = 'white';
+    activeStyle.color = 'black';
+  }
+  return (
+    <li className="page-item">
+      <a href="#" onClick={ handleClick } style={ activeStyle }>{ page }</a>
+    </li>
+  );
+};
+
+const options = {
+  pageButtonRenderer
+};
+
+export class Tickets extends Component {
+    constructor(props) {
+        super(props);
+        
+        this.state = {
+        tickets: [],
+        }
+
+        this.getTickets = this.getTickets.bind(this)
+    }    
+
+    componentDidMount() {
+        this.getTickets(this.context);
+    }
+
+    getTickets = (context) => {
+        axios.get(`${process.env.REACT_APP_API_URL}/properties`, { headers: {"Authorization" : `Bearer ${context.user.accessJwt}`} })
+        .then((response) => {
+            this.setState({properties: response.data.properties});
+        })
+        .catch((error) => {
+            alert(error);
+            console.log(error);
+        })
+    }
+
+    render() {
+        return (
+            <UserContext.Consumer>
+                {session => {
+                    this.context = session;
+                    return (
+                        <div className="tickets__container">
+                            <div className="section-header">
+                                <h2 className="page-title">Tickets</h2>
+                            </div>
+                            <div className="search-section">
+                              <input class="input is-rounded" placeholder="Search by tenant, manager, property, or JOIN staff"></input>
```

Comment:
don't forget to use "className" throughout this file instead of "class" so react can use it

Reply:
Yeah, I just copy pasted from Bulma and forgot to make the change. Sounds like we will be building out some of these primitives ourselves though